ui/src/app/applications/components/applications-list/applications-filter.tsx

```diff
@@ -28,6 +31,7 @@ export function getFilterResults(applications: Application[], pref: AppsListPref
             sync: pref.syncFilter.length === 0 || pref.syncFilter.includes(app.status.sync.status),
             health: pref.healthFilter.length === 0 || pref.healthFilter.includes(app.status.health.status),
             namespaces: pref.namespacesFilter.length === 0 || pref.namespacesFilter.some(ns => app.spec.destination.namespace && minimatch(app.spec.destination.namespace, ns)),
+            favourite: !pref.showFavourites || pref.favouritesApplist.includes(app.metadata.name),
```

Comment:
`favouritesApplist` should follow camelcase naming rule, like `favouritesAppList`.

Reply:
haha, school taught me british english - I am ok with any spelling - what do you suggest @rbreeze , @jannfis